### PR TITLE
Add support for `MixedPart` and `RelatedPart`

### PR DESCRIPTION
**Pull request by Maxoulak**
Add support for MixedPart (which can contains RelatedPart).

Useful when dealing with attachments & raw base64 images.

### CODE REVIEW

**Comment by Maxoulak** [comment]
Hi @stayallive, could you please take a look at this PR ?

**Comment by stayallive** [comment]
Hey @Maxoulak, I missed this for some reason... 

Can you please also provide some tests to prove that this code is working as intended for the features you implemented?

**Comment by Maxoulak** [comment]
Of course, I'll come back to you in few days.

File: src/SymfonyMailerCssInliner.php [hunk 1]
```diff
@@ -83,18 +85,54 @@ private function handleSymfonyMessage(Message $message): void
...
+        if ($abstractPart instanceof RelatedPart) {
+            $relatedPartParts = $abstractPart->getParts();
+
+            $mainPart = array_shift($relatedPartParts);
+
+            return new RelatedPart($this->processPart($mainPart), ...$relatedPartParts);
+        }
+
+        return null;
+    }
```
Comment (RomulusED69) [inline src/SymfonyMailerCssInliner.php:135]:
```suggestion
    }

```

**Review by RomulusED69 — CHANGES_REQUESTED** [review]

**Comment by Maxoulak** [comment]
Hi @stayallive, I've just pushed new commits including tests and fixes.
IMO, all seems good working.

**Comment by stayallive** [comment]
Hey @Maxoulak, this looks great, thanks 💪

**Pull request merged**